docs/project-overview.md

```diff
@@ -0,0 +1,83 @@
+# Project VERAISON Overview
+
+Project _VERAISON_: **VER**ific**A**t**I**on of atte**S**tati**ON**
+
+## Philosophy
+The focus of this project is to produce components that are useful in a variety of deployments.
+To that end there is a great focus on flexibility, allowing the use of plugins or policy modules to specialise the general flows that the components support.
+
+## Get in Touch
+If the following resonates with any of your plans, if you would like to contribute or if you would like to discuss the project approach in any way, please get in touch via the [Veraison Zulip channel](https://veraison.zulipchat.com)
+
+## Reference Implementations
+### Tokens
+In order that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments, many real world attestation solutions and potential extrapolations of their use have been considered. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
+To ensure that the code backs up the design, reference support for multiple tokens will be developed in the project. The current targets for this support are:
+* EAT Tokens, with the ARM PSA Profile being a specific test set. 
+* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+
+### Deployments
+To ensure that the core components are structured correctly for use as a service, the project will build reference deployments that can perform token verifications. To help ensure abstractions for the components over deployment environments are sufficiently general, multiple reference deployments will be built. They are:
+* A deployment for self hosting
+* A deployment targeting a hyperscaler PAAS environments
+Further details on the above will be available within specific documentation on the reference deployment. (ToDo) 
+
+## Scope 
+The following is the intended scope of the Veraison Project. This is being listed to help others understand the intended functionality. If you're considering adopting or contributing to Veraison and you think there's something missing for the core of an Attestation Verification service, let us know via. The following list should not be treated as a backlog, work items will be recorded elsewhere. 
```

Comment:
"let us know via" ?

Did you mean to say ZulipChat here?

Reply:
I did!